docs/user-guide/quickstart.md

```diff
@@ -308,7 +308,7 @@ shell in the client container, confirm that we can access the `s2` service runni
 in namespace `service-2`.
 
 ```bash
-kubectl run s2 -n service-2 --image=nginx --restart=Never --expose --port 8080
+kubectl run s2 -n service-2 --image=nginx --restart=Never --expose --port 80
```

Comment:
I don't think this is correct? The command on line 327 references port 8080, not port 80, and that's the only thing this needs to sync with, correct? Or is there some other reason to change this?

Reply:
I forgot to modify L327, thanks for pointing out. The reason for the modification is that the default port of nginx is 80, unless a customized nginx.conf is provided.

Another alternative is to use the official `nginxinc/nginx-unprivileged` image of nginx, which can keep the existing 8080 port example.